site/_posts/2020-09-21-troubleshooting-containerized-oidc.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "Troubleshooting OpenID-Connect Authentication on Containers"
+author: jvlcek date: 2020-09-21
+comments: true
+published: true
+tags: tutorials
+---
+
+The goal of this blog post is to provide a basic understanding of how to troubleshoot ManageIQ 
+configured for OpenID-Connect authentication when deploying ManageIQ on containers.
+
+The examples were generated using OpenShift.
+
+**Contents:**
+---------------------------------------------------------------------
+
+
+- [**Configure ManageIQ For OpenID-Connect Authentication**](#configure-manageiq-for-openid-connect-authentication)
+- [**My ManageIQ OpenID-Connect Authentication Configuration Is Broken**](#my-manageiq-openid-connect-authentication-configuration-is-broken)
+  - [**The ManageIQ Login Screen**](#the-manageiq-login-screen)
+  - [**State Of The ManageIQ Pods**](#state-of-the-manageiq-pods)
+  - [**ManageIQ Orchestrator Pod Log**](#manageiq-orchestrator-pod-log)
+  - [**ManageIQ Operator Pod Log**](#manageiq-operator-pod-log)
+  - [**ManageIQ HTTPD Pod Log**](#manageiq-httpd-pod-log)
+  - [**OpenID-Connect Provider Accessible**](#openid-connect-provider-accessible)
+  - [**Capturing User Request Headers**](#capturing-user-request-headers)
+- [**Reporting Issues**](#reporting-issues)
+- [**Known Issues**](#known-issues)
+
+# Configure ManageIQ For OpenID-Connect Authentication
+---------------------------------------------------------------------
+
+Currently Containerized ManageIQ only support the OpenID-Connect authentication configuration.
```

Comment:
I think this was really meant to be: only "supports" the  OpenID-Connect ... like it was before